### PR TITLE
fix grep string for check 1.4.11 and 1.4.12

### PR DESCRIPTION
**Pull request by hutr**
check 1.4.11 and 1.4.22 shows FAIL even when permissions is correct.

### CODE REVIEW

File: cfg/1.8/master.yaml [hunk 1]
```diff
@@ -942,9 +942,8 @@ groups:
 
   - id: 1.4.11
     text: "Ensure that the etcd data directory permissions are set to 700 or more restrictive (Scored)"
-    audit: ps -ef | grep $etcdbin | grep -v grep | sed 's%.*data-dir[= ]\([^ ]*\).*%\1%' | xargs stat -c %a
-    tests:
-      test_items:
+    audit: ps -ef | grep $etcdbin | grep -- --data-dir | grep -v grep | sed 's%.*data-dir[= ]\([^ ]*\).*%\1%' | xargs stat -c %a
+    test_items:
```
Comment (lizrice) [inline cfg/1.8/master.yaml]: Looks like this is missing `tests:`?

Comment (lizrice) [inline cfg/1.8/master.yaml]: I think this is missing `tests:`

File: cfg/1.8/master.yaml [hunk 2]
```diff
@@ -942,9 +942,8 @@ groups:
 
   - id: 1.4.11
     text: "Ensure that the etcd data directory permissions are set to 700 or more restrictive (Scored)"
-    audit: ps -ef | grep $etcdbin | grep -v grep | sed 's%.*data-dir[= ]\([^ ]*\).*%\1%' | xargs stat -c %a
-    tests:
-      test_items:
+    audit: ps -ef | grep $etcdbin | grep -- --data-dir | grep -v grep | sed 's%.*data-dir[= ]\([^ ]*\).*%\1%' | xargs stat -c %a
```
Comment (lizrice) [inline cfg/1.8/master.yaml]: I don't think there is any need for the grep -v grep step now that we are checking for --data-dir

File: cfg/1.8/master.yaml [hunk 3]
```diff
@@ -960,7 +959,7 @@ groups:
 
   - id: 1.4.12
     text: "Ensure that the etcd data directory ownership is set to etcd:etcd (Scored)"
-    audit: ps -ef | grep $etcdbin | grep -v grep | sed 's%.*data-dir[= ]\(\S*\)%\1%' | xargs stat -c %U:%G
+    audit: ps -ef | grep $etcdbin | grep -- --data-dir | grep -v grep | sed 's%.*data-dir[= ]\(\S*\)%\1%' | xargs stat -c %U:%G
```
Comment (lizrice) [inline cfg/1.8/master.yaml]: I think grep -v grep  is now unnecessary here too

File: cfg/1.8/master.yaml (hunk 1 above)
Comment (hutr) [reply cfg/1.8/master.yaml]: Added tests:

File: cfg/1.8/master.yaml (hunk 2 above)
Comment (hutr) [reply cfg/1.8/master.yaml]: You are correct about this. Removed grep -v grep

File: cfg/1.8/master.yaml (hunk 3 above)
Comment (hutr) [reply cfg/1.8/master.yaml]: Removed grep -v grep

Thanks for correction!

**Comment by hutr** [comment]
Thanks for the correction.

Added tests: for 1.4.11
Removed grep -v grep for 1.4.11 and 1.4.12
Also fixed sed string for 1.4.12 to fetch right --data-dir folder

**Pull request merged**